Database_Restructure/DB_restructure_documentation.md

```diff
@@ -0,0 +1,129 @@
+
+The DS team would repeat this process for the other collections as indicated by the design, and once Web had connected their forms and data pipelines to the MongoDB database (and disconnected their forms and pipelines from the duplicate tables in Postgres), the Web team would be responsible for deleting all non-authentication tables from the Postgres database.
+
+## Current State
+
+There is a current design for what collections will be held in the `schema.py` file in the DS codebase (mongodb_schema_design.xls). It currently contains 5 collections:
+- Mentor
+- Mentee
+- Meeting
+- Resource
+- Ticket
+
+Mentor, Mentee, and Meeting have:
+Create class added to `schema.py`, update class added to `schema.py`, data generators created in `generators.py`, changes operate with `seeds.py` and API calls (to the best knowledge of the DS team).
+
+Resource has a class in `schema.py` but it has not been updated with changes. **NOTE:** There is also a 'Resources' class, which is residual from the Postgres table mirroring.
+
+Ticket does not have a class. **NOTE:** There is a TicketsTable class which appears to have the fields for the Ticket collection in the new design, but it does not have an update class.
+
+There are other collections in the `schema.py` file not discussed thus far:
+
+- The Feedback class, as well as its Update and Options class, are currently used for development purposes to build DS models and visualizations (NLP, vader sentiment score, etc.).
+- There are still a number of collections which are residuals from the Postgres mirroring: MentorIntake, Role, Comments, Notes, MenteeProgression, Assignment, Reviews
+
+
+
+## Future Needs and Considerations (Roadmap)
+
+### Current Collections
+
+The Mentor, Mentee, and Meeting collections are consistent with the needs of the minimum viable product (MVP). Resource is a stretch goal.
+
+The changes to the Mentor and Mentee collections have introduced bugs in the frontend/backend logic. This is currently being addressed, but as of now is yet to be completed.
+
+There is a lot of work to be done by frontend for collecting and displaying meeting information. There is a calendar displayed on the website but there do not appear to be any Meeting forms for users.
+
+### Ticket
+
+Ticket collection is needed for MVP but it is undecided whether this will be a single collection or a set of collections that will all appear as tickets on the website. There are two approaches (so far) and one will need to be chosen before creating the collections:
+
+1. All tickets are consolidated into a single collection and the type of ticket is indicated by a `ticket_type` with options (which may grow if new ticket types are added) `['escalation', 'application', 'resource']`. 
+2. Ticket types are separated into unique collections and are consolidated into tickets displayed on the website via backend/frontend logic.
+
```

Comment:
I'm excited to be involved in this discussion--since there are now WEB people who can give input.